zqd/space/manager.go

```diff
@@ -86,15 +99,34 @@ func (m *Manager) Create(name, dataPath string) (*Space, error) {
 		return nil, errors.New("created duplicate space id (this should not happen)")
 	}
 
-	sp, err := loadSpace(path, c)
+	spaces, err := loadSpaces(path, c)
 	if err != nil {
 		return nil, err
 	}
-	m.spaces[id] = sp
-	return sp, nil
+	if len(spaces) != 1 {
+		panic("multiple spaces created during space create")
+	}
+	m.spaces[id] = spaces[0]
+	return spaces[0], nil
 }
 
-func (m *Manager) Get(id api.SpaceID) (*Space, error) {
+func (m *Manager) SubspaceCreate(parent Space, req api.SubspacePostRequest) (Space, error) {
```

Comment:
Nit: Call this CreateSubspace for consistency with (*archiveSpace).CreateSubspace.